influxdb/README.md

```diff
@@ -0,0 +1,18 @@
+InfluxDB 2.0 interpreter for Apache Zeppelin
+============================================
+
+## Description:
+
+Provide InfluxDB Interpreter for Zeppelin.
+
+## Build
+
+```
+mvn -pl zeppelin-interpreter,zeppelin-display,influxdb -DskipTests package
```

Comment:
You can use `mvn -pl influxdb -am -DskipTests package`, influxdb depends on zeppelin-interpreter-shaded, so use `-am` would be easier. BTW, does influxdb interpreter depends on zeppelin-display ?

Reply:
The interpreter does not depends on zeppelin-display, I just copied it from another interpreter. It is very basic interpreter now. In future we would like to add syntax highlighting and code complete.